code-challenges.test.js

```diff
@@ -13,39 +13,101 @@
-// Expected output: [72, 81, 90, 99, 108]
-
-// b) Create the function that makes the test pass.
-
-// Pseudo code:
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
+
+// // b) Create the function that makes the test pass.
+
+// describe('multipliedByThree', () => {
+//     it('takes in an array of numbers and returns an array with all the numbers multiplied by 3', () => {
+//       const numbersArray1 = [6, 7, 8, 9, 10]
+//       // Expected output: [18, 21, 24, 27, 30]
+//       const numbersArray2 = [24, 27, 30, 33, 36]
+//       // Expected output: [72, 81, 90, 99, 108]
+//       expect(multipliedByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+//       expect(multipliedByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+//     })
+//   })
+
+// // Pseudo code:
+// // input: array of numbers
+// // output: array with all numbers multiplied by three
+// const multipliedByThree = (arr) => {
+//     // iterate using .map
+//     return arr.map((value) => {
+//       // multiply current value of each iteration by 3
+//       // return product in the new array created by .map
+//       return value * 3
+//     })
+//   }
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("divisibleByThree", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+        const object1 = { number: 15 }
+        const object2 = { number: 0 }
+        const object3 = { number: -7 }
+        // Expected output: "15 is divisible by three"
+        // Expected output: "0 is divisible by three"
+        // Expected output: "-7 is not divisible by three"
+        expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
+
+// ReferenceError: divisibleByThree is not defined
+
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// // Pseudo code:
+const divisibleByThree = (object) => {
+    // iterate with if statement using modulo 3
+    // compare remainder of each object to 0
+    if (object.number % 3 === 0) {
+        return `${object.number} is divisible by three`
+        // return if object number is divisible by three or not
+      } else {
+        return `${object.number} is not divisible by three`
+      }
+}
```

Comment:
if statements don't start iteration. I like the line by line pseudo code, I think you just need to take an extra minute to think about what you're writing and see if it's actually exactly what the code is doing or not.